src/util/user.h

```diff
@@ -24,6 +25,21 @@ enum {
         _USER_SLOT_N,
 };
 
+static inline const char *user_slot_to_string(size_t slot) {
+        switch (slot) {
+        case USER_SLOT_BYTES:
+                return "bytes";
+        case USER_SLOT_FDS:
+                return "FDs";
+        case USER_SLOT_MATCHES:
+                return "matches";
+        case USER_SLOT_OBJECTS:
+                return "objects";
+        default:
+                assert(0);
+        }
+}
+
```

Comment:
Why is that in the header?

Reply:
So far we kept the slot constants in the header with a TODO to move them elsewhere, so I kept this together.